app/build.gradle

```diff
@@ -96,4 +101,16 @@ task zipMagiskModule(type: Zip) {
 	archiveName zipName
 	destinationDir outDir
 	dependsOn(":app:assembleMagiskModule")
+}
+
+task pushMagiskModule(type: Exec) {
+	commandLine('adb', 'push', zipFile.absolutePath, "/sdcard/Documents/" + zipName)
+	dependsOn(":app:zipMagiskModule")
+}
+
+task testMagiskModule(type: Exec) {
+	commandLine('adb', 'shell', 'su', '-c',
+			"magisk --install-module /sdcard/Documents/" + zipName +
+					" && (/system/bin/svc power reboot || /system/bin/reboot)")
```

Comment:
should use `(stop && start)` command

Reply:
Magisk modules only apply on full system reboot.
I don't think stop/start command will apply Magisk modules updates.

Reply:
Oh, good point